docs/iis.md

```diff
@@ -2,17 +2,50 @@
 
 Follow the [installation guide](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/iis) for instructions to collect metrics from this application using Ops Agent.
 
-## Metrics
+Additionally, specify `receiver_version` as `v2` to enable new metrics gathering. 
 
-The following table provides the list of metrics that the Ops Agent collects from this application.
+Example `v2` Configuration
+```yaml
+metrics:
+  receivers:
+    iis:
+      type: iis
+      receiver_version: "2"
```

Comment:
There's no need to quote — looks like the `yaml` library will do the right thing here: https://go.dev/play/p/NCqHS7RwnKF.

Reply:
Updated